frontend/src/components/Basics/_typography.scss

```diff
@@ -7,9 +7,11 @@
   @if $size == 1 {
     font-size: 3rem;
     line-height: 3.625rem;
+    margin: 0.67em 0; // Temporary: added margin for current components
   } @else if $size == 2 {
     font-size: 2.25rem;
     line-height: 2.75rem;
+    margin: 0.83em 0; // Temporary: added margin for current components
```

Comment:
So the TW preflight styles override all of the styles in `_normalize.scss` right ?

But moving the styles into this file overrides the TW preflight?

Reply:
I believe so. It's not ideal--I think the standard is to remove all styles for headings in the normalized css, but I didn't want our demo page to look too different from before so I added these margins only for h1 and h2 in _typography.scss, but by doing so, new components/pages will also have h1 and h2 with these margins. My hope is that in the near future we can update component styles so we can 100% rely on TW preflight. What are your thoughts on this?

Reply:
I think its a question of tailwind philosophy that we should discuss. My understanding is that with tailwind you are supposed to apply absolutely every style with classNames right? 

so if we wanted a global h2 style the only option that adheres to tailwind best practices is to create a component just for h2 styles that we would then import anytime we wanted use an h2 element?

or we could just run an extra global stylesheet in addition to tailwind. which do you prefer?

Reply:
Maybe it would be better to collect any normalization conflict fixes in the `index.scss` file ?

Reply:
That makes sense. I'll move the styles.